src/util.js

```diff
@@ -66,12 +66,13 @@ function getBestSelectionFromSrcset (el, scale) {
   const result = []
   const container = el.parentNode
   const containerWidth = container.offsetWidth * scale
+  const splitArg = options.includes(',\n') ? ',\n' : ', '
```

Comment:
Should `splitArg` be `/,\s*/` ?

Reply:
I tried with `/,\s+/` and ain't working.  With the `,\n` works fine.

Reply:
Why the `/,\s+/` does not work? `/,\n/` is not very suitable.